src/index.mjs

```diff
@@ -86,6 +90,24 @@ export default class SizePlugin {
 		compiler.hooks.afterEmit.tapPromise(NAME, compilation => this.outputSizes(compilation.assets).catch(console.error));
 	}
 
+  async storeToFile (buildResult) {
+    try {
+      fs.readFile(`${this.historyPath}`, (err, data) => {
+        let json = {};
+        if (data) {
+          json = JSON.parse(data);
+        }
+        json[this.buildTimestamp] = buildResult[this.buildTimestamp];
+        fs.writeFile(`${this.historyPath}`, JSON.stringify(json), (e) => {
+          if (e) throw e;
+          console.log(chalk.green(`file got stored at: ${this.historyPath}`));
+        });
+      })
+    } catch (e) {
+      throw e;
+    }
+  }
```

Comment:
I think its good to `promisify` all `fs` methods so  we can make use of async/await function,
let me know your view :+1:

Reply:
Good point i made some changes, i didn't use promisify util to make sure it can still support older versions of node. I've also updated the readme file to show the option let me know what you think